.travis.yml

```diff
@@ -20,6 +20,10 @@ addons:
       - libseccomp-dev
 
 before_install:
+  # the first 3 lines enable running travis on personal github repositories
+  - cd ../..
+  - if [ ! -d singularityware ]; then mv * singularityware; fi
+  - cd singularityware/singularity
```

Comment:
Heya @DrDaveD , could you update this to `sylabs/singularity` ;) 
to update to recent changes.

Reply:
Done